client-runtime/src/main/java/com/microsoft/rest/v3/http/NettyClient.java

```diff
@@ -26,13 +25,12 @@
 import java.util.Map;
 import java.util.Objects;
 import java.util.function.BiFunction;
-import java.util.function.Function;
 
 /**
  * An HttpClient that is implemented using Netty.
  */
 public final class NettyClient extends HttpClient {
-    private final HttpClientConfiguration configuration;
+    private reactor.netty.http.client.HttpClient httpClient;
```

Comment:
final

Reply:
Yuck. Can't do. Reactor creates a new instance when configuring the TCP configurations.